koa-crud/test/integration/use-cases/vendors.spec.ts

```diff
@@ -149,7 +149,9 @@ describe('Vendor Use Cases', () => {
     it('should return list of vendors', async function () {
       await expect(
         selectAllVendorsUseCase({ id: null, info: null, source: null }),
-      ).to.eventually.fulfilled.and.length(1);
+      )
+        .to.eventually.fulfilled.and.property('totalCount')
```

Comment:
since you are only testing `totalCount` you can do this: `property('totalCount', 1)`

Reply:
Noted on this